conf/script/build-ha-io

```diff
@@ -738,8 +738,10 @@ statsd_rsc_add() {
 
 haproxy_rsc_add() {
     log "${FUNCNAME[0]}: Adding haproxy to pacemaker..."
-    sudo pcs -f $cib_file resource create haproxy-c1 systemd:haproxy op monitor interval=30s
-    sudo pcs -f $cib_file resource create haproxy-c2 systemd:haproxy op monitor interval=30s
+    sudo pcs -f $cib_file resource create haproxy-c1 systemd:haproxy op monitor interval=30s \
```

Comment:
Repeated failures means something is wrong with haproxy and needs manual intervention. Does clearing failure count automatically means keep on trying infinitely? If yes, then why do we need it and what is the case it's handling?

Reply:
>Does clearing failure count automatically means keep on trying infinitely?
haproxy resources are active/active and anyway are confined to a node, pacemaker anyway tries to restart a failed resource, failure-timeout will just help to clear the failcount and avoid the resource to be banned if failcount reaches 100000 (which high, but can occur during testing, as in case of EOS-13296).

So, if there's some problem with haproxy (e.g. configuration or dependency) itself then we must look into it. But say if haproxy started but failed for some reason (some transient network issue, minor configuration problem post startup), pacemaker does try to restart it anyway but the failcount is not cleared, setting failure-timeout will clear the failcount as well, typically the failcount has to reach 100000 for the resource to be banned.
Note that start/stop failures are still fatal, i.e. pacemaker will set the failcount to -INFINITY if a resource fails to start/stop, this is not cleared by failure-timeout automatically and requires some additional global properties to be set, start-failure-is-fatal (default true) or cluster-recheck-interval (default 15 mins). I think we should not touch the global properties yet.